storage/options.go

```diff
@@ -147,6 +147,7 @@ func newOptions(poolOpts pool.ObjectPoolOptions) Options {
 	})
 	bytesPool.Init()
 	seriesOpts := series.NewOptions()
+	pm, _ := fs.NewPersistManager(fs.NewOptions())
```

Comment:
could you check error here and add a `Validate()`/bubble up/panic for it.

Reply:
Yeah =[  

It's going to be not ideal because will have to cache the error (to be able to return it from Validate) and then return it if its not nil or something like that.  

Will do.